### ISSUE TITLE
Stay on top in multi-monitor and fullscreen

### ISSUE THREAD

**Issue by Richard-L** [OpenRCT2/OpenRCT2#3227]
Perhaps a bit specific, but in a multi-monitor setup and running ORCT2 in fullscreen on screen, can it stay on top even if a different screen is active? Effecitvely keep showing the game while you're typing elsewhere, how it would be in windowed mode.

**Comment by janisozaur** [comment]
I thought that was the purpose of this option:

![zaznaczenie_109](https://cloud.githubusercontent.com/assets/550290/14229418/d9e80646-f933-11e5-8bbf-73d31f80c98a.png)

**Comment by Richard-L** [comment]
Nope. Screen returns back to Desktop while game keeps running in the back.

And is this an older screenshot? This option is called "Fullscreen (borderless)" for me.

**Comment by Gymnasiast** [comment]
It is, yes. The terminology was changed to make it a bit clearer.

**Comment by IntelOrca** [comment]
Its actually because the US version hasn't been updated yet.

**Comment by Gymnasiast** [comment]
Er, no. Richard-L's uses the new (updated) terminology and I very much doubt that janisozaur would use US English rather than UK. So most likely an old screenshot. (Not terribly important, though.)

**Comment by AaronVanGeffen** [comment]
Looks like [it hasn't](https://github.com/OpenRCT2/OpenRCT2/blob/develop/data/language/english_us.txt#L2785), though.

**Comment by Richard-L** [comment]
In any case, on-top fullscreen display would be so useful, so please do consider!

**Comment by Gymnasiast** [comment]
@AaronVanGeffen : You're missing the point. US English hasn't been updated, but neither Richard-L nor janisozaur use it.

**Comment by IntelOrca** [comment]
@Gymnasiast Are you saying that @janisozaur has not kept his language files up to date? When he said he symlinks his data folder to the git repo one.

**Comment by janisozaur** [comment]
Guys! stay on topic. I'm not sure why the screen has outdated text in it, I must've been using unmerged/stale/not-yet-rebased branch, but it says the proper thing now. This is irrelevant.
